includes/cf/dv-info-green-purple.md

```diff
@@ -0,0 +1,17 @@
+??? info "Why am i getting purple or green colors - [Click to show/hide]"
+
+    Why am i getting purple or green colors?
+
+    There could be several reasons why your TV shows purple or green colors when playing Dolby Vision content. Here are some possible explanations:
+
+    1. Unsupported Hardware: Your TV or hardware media player device (Roku, AppleTV, etc) doesn't support Dolby Vision. Or your hardware media player device doesn't support all the Dolby Vision Profiles. As a result, the device might struggle to produce accurate colors, leading to a purple or green tint.
+
+    1. Incorrect Display Settings: Dolby Vision content often requires specific settings on the TV or display device to be enabled in order to deliver the intended visual experience. If these settings are not configured correctly, it can result in the device showing distorted colors, including purple or green hues.
+
+    1. HDMI Compatibility Issues: Sometimes, HDMI cables or ports may not be fully compatible with Dolby Vision. If the media player device is not recognizing the Dolby Vision signal properly, it may fail to process the content correctly, resulting in abnormal color rendering.
+
+    To resolve the purple or green color issues when playing Dolby Vision content, you can try the following troubleshooting steps:
+
+    1. Ensure your TV or hardware media player device is Dolby Vision compatible and up-to-date with the latest firmware.
+    1. Verify that your TV or display device is set up correctly and has the necessary Dolby Vision settings enabled.
+    1. Check the HDMI cables and ensure they are capable of transmitting Dolby Vision signals.
```

Comment:
```suggestion
??? info "Why am I getting purple or green colors? - [Click to show/hide]"

    Why am I getting purple or green colors?

    There are several possible reasons why your TV would show purple or green colors when playing Dolby Vision content.

    1. **Unsupported Hardware**: Your TV or hardware media player device (Roku, AppleTV, etc) doesn't support Dolby Vision or your hardware media player device doesn't support all the Dolby Vision Profiles. As a result, the device might struggle to produce accurate colors, leading to a purple or green tint.

    1. **Incorrect Display Settings**: Dolby Vision content often requires specific settings to be enabled on your TV or display device in order to deliver the intended visual experience. If these settings are not configured properly, it can result in the device showing distorted colors (including purple or green hues).

    1. **HDMI Compatibility Issues**: Sometimes, HDMI cables or ports may not be fully compatible with Dolby Vision. If the media player device is not recognizing the Dolby Vision signal properly, it may fail to process the content correctly, resulting in abnormal color rendering.

    To resolve the purple or green color issues when playing Dolby Vision content, you can try the following troubleshooting steps:

    1. Ensure your TV or hardware media player device is Dolby Vision compatible and up-to-date with the latest firmware.
    1. Verify that your TV or display device is set up correctly and has the necessary Dolby Vision settings enabled.
    1. Check the HDMI cables and ensure they are capable of transmitting Dolby Vision signals.
```